operator/cmd/mesh/install.go

```diff
@@ -204,7 +204,7 @@ func InstallManifests(setOverlay []string, inFilenames []string, force bool, dry
 		return nil, err
 	}
 
-	if err := createNamespace(clientset, iop.Namespace, networkName(iop)); err != nil {
+	if err := createNamespace(clientset, v1alpha12.Namespace(iop.Spec), networkName(iop)); err != nil {
```

Comment:
will this also handle `values.global.istioNamespace` ?

Reply:
Yes, see: https://github.com/istio/istio/blob/5ae68fb2df850a13bebba2533b25279c956dd511/operator/pkg/apis/istio/v1alpha1/common.go#L29-L46